src/main/java/ekud/command/ListCommand.java

```diff
@@ -0,0 +1,76 @@
+package ekud.command;
+
+import java.time.LocalDate;
+
+import ekud.common.exception.EkudException;
+import ekud.storage.Storage;
+import ekud.task.Task;
+import ekud.task.TaskList;
+import ekud.task.TaskWithDateTime;
+
+/**
+ * Command that lists all tasks, optionally filter by date if given.
+ */
+public class ListCommand extends Command {
+    private final boolean filter;
```

Comment:
Could this variable be named to sound more 'boolean'? (I did like that you've followed boolean naming conventions in other places)